dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/preheat/supplier/EnrollmentSupplier.java

```diff
@@ -73,4 +70,13 @@ public void preheatAdd(TrackerObjects trackerObjects, TrackerPreheat preheat) {
           });
     }
   }
+
+  private List<Enrollment> getEnrollmentsByProgram(List<Program> programs) {
```

Comment:
This is not strictly related to this PR but could we make the query just get all the enrollments with programs without registration instead of getting the programs and passing the ids?